grammar.js

```diff
@@ -770,7 +794,7 @@ module.exports = grammar({
 
     type_parameter: $ => seq(
       'type',
-      $.type_identifier,
+      repeat($.type_identifier),
```

Comment:
What’s the purpose of this `repeat`?

Reply:
[Renalyze examples](https://github.com/rescript-association/reanalyze/blob/a3f9bdedf0431f7dbc3422ad8e9f12df155192e1/examples/deadcode/src/TestFirstClassModules.res#L26-L30)

```res
@genType
let convertFirstClassModuleWithTypeEquations = (
  type o i,
      // ^ here
  x: module(MT with type Inner.inner = i and type outer = o),
) => x
```

Reply:
Changed to `repeat1` e0522d33d4dcc5426edaa05493e0c14b7e8ca141